### PR TITLE
Branding settings: Login Background & Banner

### PR DESCRIPTION
**Pull request by torchiaf**
### Summary
Fixes https://github.com/rancher/dashboard/issues/10423
Bocked by: https://github.com/rancher/rancher/issues/44577

### Occurred changes and/or fixed issues

We add new settings to customize the Login Background and Dashboard's Homepage banner.

### Technical notes summary

New settings required:

1. `ui-login-background-light` - for the background, light theme image
2. `ui-login-background-dark` - for the background, dark theme image
3. `ui-banner-light` - for the banner, light theme image
4. `ui-banner-dark` - for the banner, dark theme image

### How to test

- Run Rancher server locally, using this [branch](https://github.com/rancher/rancher/pull/44961)
  - https://github.com/rancher/rancher/wiki/Setting-Up-Rancher-2.0-Development-Environment#set-up-vscode
  or
  - See internal documentation

- Run a Rancher UI locally - `API=https://127.0.0.1:8443 yarn dev`

### Areas or cases that should be tested

- General Settings -> Branding page
- Login Background
- Home Page -> banner on top

### Areas which could experience regressions

- Settings -> Branding page

### Screenshot/Video

Banner in Homepage

image

Branding tab

image

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [x] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

**Review by yonasberhe23 — COMMENTED** [review]
e2e tests look solid to me, nice work! i'll defer to @richard-cox to review the feature code

File: shell/assets/translations/en-us.yaml [hunk 1]
```diff
@@ -7312,6 +7312,22 @@ branding:
...
+    label: Banner
+    tip: 'Upload an image to replace the Rancher banner in the Dashboard page. Max file size is 200KB'
+    lightPreview: Light Theme Preview
+    darkPreview: Dark Theme Preview
+    uploadLight: Upload Light Banner
+    uploadDark: Upload Dark Banner
+    useCustom: Use a Custom Banner
+  loginBackground:
+    label: Login Background
+    tip: 'Upload an image to replace the Rancher background in the login page. Max file size is 200KB'
```
Comment (rak-phillip) [inline shell/assets/translations/en-us.yaml:7325]: Do we want to give any hints to dimensions that might work best for the banner and login background? I see that we already do this for the logo.

**Review by rak-phillip — COMMENTED** [review]
I'm testing this with two images that I selected for the banner and the login background:

custom-banner

I can see the banner used on the home page

home-page

But I don't see my custom login image in use

login-page

Are there restrictions in place for the login image that aren't being relayed to me?

File: shell/assets/translations/en-us.yaml (hunk 1 above)
Comment (torchiaf) [reply shell/assets/translations/en-us.yaml:7325]: Good catch, thanks. @kwwii do you a tip on how to calculate the recommended dimensions for banners and login background ?

**Review by rak-phillip — APPROVED** [review]
LGTM - I think the only thing that we'd like to figure out is if we'd like to update the description.

Edit: The previous issue with the login image appears to have been an issue with my environment and not related to this change.

**Review by rak-phillip — DISMISSED** [review]

**Comment by torchiaf** [comment]
I changed again the image tips. Could you approve it again @rak-phillip  ?

**Review by rak-phillip — APPROVED** [review]

**Pull request merged**